### PR TITLE
Revision files added

### PR DESCRIPTION
**Pull request by rimanu**
New files for importing and splitting the real data sets used in the numerical experiment, computing the reference models KronRLS and KronSVM for the real data sets and executing the LMBM-KronL0LS algorithm with both main objectives and starting point procedures for the real data sets.

### CODE REVIEW

File: load_split_data.py [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+    Y_log = np.log(Y)
+    return XD, XT, Y_log, drug_inds.astype(int), target_inds.astype(int), Y_binary
+
+"""
+Function to load an incomplete data set introduced by Merget et al. (2017) and updated by Cichonska et al (2018).
+Returns the data matrices and lists of drug and target indices for the known pairs.
+The matrices of drug similarities and target similaritied are multiplied by 100 in order to 
+obtain the same range as in the corresponding matrices of Metz data.
+"""
+def load_merget(threshold = 7):
```
Comment (pavetsu14) [inline load_split_data.py:43]: Should these files also be added to the repository or are they classified?

**Review by pavetsu14 — COMMENTED** [review]
I added one comment, please take a look at it.

Otherwise; thank you for the heavy refactoring of the code in addition to making the necessary changes for the revision!

File: load_split_data.py [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+    S3_test_validation = [S3_test_indices, S3_validation_indices]
+    S4_test_validation = [S4_test_indices, S4_validation_indices]
+    return(df_indices, training_indices, S1_test_validation, S2_test_validation, S3_test_validation, S4_test_validation)
+
+if __name__ == "__main__":
+    # Select a seed or multiple seeds for controlling the randomness in creating the splits. 
+    random_seeds = [2688385916]
+    # Choose the percentage of drugs and targets that are defined to be in area S1 and the percentage of area S1 pairs that will be used as training data.
+    split_percentage = 0.6
+    datasets = ["davis", "metz", "merget"]
```
Comment (pavetsu14) [inline load_split_data.py:157]: Previously I also calculated tests for chessboard data. Could you add it also in here (see L0ridge.py file and generate_chess function)? Or have we decided to exclude these tests from the revision version of the article?

**Comment by rimanu** [comment]
Chessboard calculations are not rerun during the revision, they are left as
they were previously. I think it is less confusing to add links directing
to where the data sets can be found than add all of them in the repository.
I need to ask if Merget data is such that it can be publicly available.

ma 24. lokak. 2022 klo 8.43 Pauliina Paasivirta ***@***.***>
kirjoitti:

**Comment by pavetsu14** [comment]
I see. In that case I might change L0ridge.py file only to contain chessboard data tests. All the tests with real world data would then be in your revision file.

Let me know what is the case with Merget data. According to open source principles everything needed to run the code should be located in the repository. If the codes contain references to classified data, it should be at least stated in the ReadMe.

**Comment by pavetsu14** [comment]
After taking a look at these revision files I felt a need to modify the old codes to better match with the files you made. I created an issue of all the things bothering me and created a PR for resolving it. I also merged it and thus main branch is now ahead of this feature branch. Please run
`git rebase origin/master`
in order to get your branch up to date.

No merge conflicts should appear since I did not touch any of the files in this PR. I just deleted a bunch of unnecessary files, renamed a couple of files and modified old L0ridge.py file so that it contains only codes needed to run the method for Checker data.

Now all the tests with real world data are in these your revision files.

**Pull request merged**